### PR TITLE
DAT-78969 select portal dynamic width

### PR DESCRIPTION
**Pull request by mlevi1806**
## Description
select portal dynamic width
## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Are you adding a new package?
- [ ] yes
- [x] no

### CODE REVIEW

File: src/components/Select/SelectMenu.js [hunk 1]
```diff
@@ -238,6 +228,8 @@ const PortalSelectContainer = styled.div.attrs(({ xPos, inlineSearch }) => ({
 }))`
   ${({ theme }) => theme.animation.fade}
   top: ${({ theme, yPos }) => `calc(${yPos}px + ${theme.size.MEDIUM})`};
+  left: ${({ xPos }) => `${xPos}px`};
```
Comment (Chikel) [inline src/components/Select/SelectMenu.js:231]: Isn't it the same as line 225?
And aren't we override width?

**Review by Chikel — CHANGES_REQUESTED** [review]

File: src/components/Select/SelectMenu.js (hunk 1 above)
Comment (mlevi1806) [reply src/components/Select/SelectMenu.js:231]: Yep, something is messed up while resolving conflicts

**Review by Chikel — APPROVED** [review]

**Pull request merged**